CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
-# Changelog
+
+gelog# Changelog
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [7.0.6]
+
+- Add .rdmp extention
```

Comment:
Nope, 7.0.5 had an .rdmp plugin package as well ...
+ `- Update for changed RDMP LoadMetaData API`